docker-for-ibm-cloud/load-balancer.md

```diff
@@ -74,62 +84,71 @@ on that port to your service.
 >
 > Each cluster's service load balancer can have 10 ports opened. If you create new services or update a service to publish it on a port but already used 10 ports, new ports are not added and the service cannot be accessed through the load balancer.
-> If you need more than 10 ports, you can explore alternative solutions such as [UCP domain names](https://docs.docker.com/datacenter/ucp/2.2/guides/admin/configure/use-domain-names-to-access-services/) or [Træfik](https://github.com/containous/traefik). You can also [create another cluster](administering-swarms.md#create-swarms).
+> If you need more than 10 ports, you can explore alternative solutions such as [UCP domain names](/datacenter/ucp/2.2/guides/admin/configure/use-domain-names-to-access-services/) or [Træfik](https://github.com/containous/traefik). You can also [create another cluster](administering-swarms.md#create-swarms).
+
+To learn more about general swarm networking, see the [Docker container networking](/engine/userguide/networking/) and [Manage swarm service networks](/engine/swarm/networking/) guides.
 
-### Accessing a service with the service load balancer
+### Access a service with the service load balancer
+
+Get a publicly accessible HTTP URL for your app by publishing a Docker service on an unused port. For secure HTTPS URLs, see [Services with SSL certificates](#services-with-ssl-certificates).
 
 1. Connect to your Docker EE for IBM Cloud swarm. Navigate to the directory where you [downloaded the UCP credentials](administering-swarms.md#download-client-certificates) and run the script. For example:
 
    ```bash
    $ cd filepath/to/certificate/repo && source env.sh
    ```
 
-2. Create the service specifying the port on which you want the service exposed. For example:
+2. Create the service that you want to expose by using the `docker service create` [command](/engine/reference/commandline/service_create/). For example:
 
    ```bash
-   $ docker service create --name go-demo \
-     -e DB=go-demo-db \
-     --network go-demo \
-     --publish 8080:8080 \
-     vfarcic/go-demo
+   $ docker service create --name nginx-test \
+     --publish 8080:80 \
+     --replicas 3 \
+     nginx
    ```
 
-3. Get the name of the cluster, and then use it to get the service load balancer URL:
+3. List the name of the cluster, and then use it to show the service `svc` load balancer URL:
 
    ```bash
    $ bx d4ic list --sl-user user.name.1234567 --sl-api-key api_key
-   $ bx d4ic show --swarm-name my_swarm --sl-user user.name.1234567 --sl-api-key api_key
+   $ bx d4ic show --swarm-name mycluster --sl-user user.name.1234567 --sl-api-key api_key
    ```
 
-4. To access a service that you have previously exposed on a port, use the `service-lb-url` that you retrieved. For example:
+4. To access the service that you exposed on a port, use the service `svc` load balancer URL that you retrieved. The load balancer might need a few minutes to update. For example:
```

Comment:
I don't understand what `svc` is doing here...

Reply:
`svc` is the abbreviation for service that's used in the CLI, so I've changed it to bold and put in brackets to hopefully clarify this relationship.